apps/kyc-server/src/components/dashboard/table/user-table.tsx

```
@@ -0,0 +1,113 @@
+
+	if (error) {
+		return (
+			<div className="flex w-full flex-col justify-start gap-6">
+				<div className="flex items-center justify-center h-64 px-4 lg:px-6 rounded-lg border border-red-200 bg-red-50">
+					<div className="text-center space-y-4">
+						<p className="text-red-600 font-medium">Failed to load users</p>
+						<p className="text-red-500 text-sm">{error}</p>
+						<button
+							onClick={actions.refetch}
+							className="px-4 py-2 text-sm text-white bg-red-600 hover:bg-red-700 rounded-md transition-colors"
+						>
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)